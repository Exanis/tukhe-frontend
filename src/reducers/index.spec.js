import store from './index';

it('Should configure store as expected', () => {
    expect(store).toMatchSnapshot()
});
