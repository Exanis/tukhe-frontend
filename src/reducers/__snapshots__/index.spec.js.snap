// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should configure store as expected 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
